modules/lib/src/main/resources/assets/js/app/view/context/widget/version/ContentVersionsConverter.ts

```diff
@@ -119,6 +156,21 @@ export class ContentVersionsConverter {
         return createParams;
     }
 
+    private getPreviousVersion(version: ContentVersion): ContentVersion {
+        let previousVersion: ContentVersion = null;
+
+        this.filteredVersions.some((v: ContentVersion, index: number) => {
+            if (version === v) {
+                previousVersion = this.filteredVersions[index + 1];
```

Comment:
Did you test the case when `version === this.filteredVersions[this.filteredVersions.length-1]`? In this case `previousVersion = this.filteredVersions[index + 1];` will give `undefined` (or even an exception).

Reply:
it will give undefined which is totally ok in this situation